src/components/site/SummaryList.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const SummaryList = props => {
+  const { items = [] } = props;
+  return (
+    <React.Fragment>
```

Comment:
this should be `<div className="dg_summary-list">`

Reply:
done